AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -35,7 +35,7 @@
  */
 @interface ASStackLayoutSpec : ASLayoutSpec
 
-/** Specifies the direction children are stacked in. */
+/** Specifies the direction children are stacked in. It becomes immutable after either horizontal or vertical alignment was set */
```

Comment:
And since the restriction is not related to the problem at hand, I think it deserves another PR.